README.md

```diff
@@ -1,29 +1,59 @@
+
+Accepted records are summarised twice daily. These summaries can be accessed with a GET request to `.../api/v1/cloud/record/summary`. Summaries can be filtered using `key=value` pairs. See [Supported key=value pairs](doc/user.md#supported-keyvalue-pairs) for a list of valid supported `key=value` pairs. A GET request requires a IAM access token be included in the request. This token is then sent to the IAM to authenticate the ID of the service requesting access to the summary. This ID needs to be in `ALLOWED_FOR_GET` in `apel_rest/settings.py` for access to be authorized. See [Authorize new WP5 components to view Summaries](doc/admin.md#authorize-new-wp5-components-to-view-summaries) for instructions on adding service to `ALLOWED_FOR_GET`
+
+It is currently expected that only the QoS/SLA tool will interact with these summaries.
+
+### Features of First Release (Version 1.0.0-1)
+
+- Accept APEL-Cloud v0.2 usage records via POST requests to the REST end point `.../api/v1/cloud/record`
+- Provide access to summaries via GET requests to REST end point `.../api/v1/cloud/record/summary`
+
+## Running the docker image on Centos 7 and Ubuntu 14.04
+We recommend using the docker image to run the Accounting server and REST interface. As such, having Docker installed is a prerequisite.
+
+See https://docs.docker.com/engine/installation/linux/ubuntulinux/ or https://docs.docker.com/engine/installation/linux/centos/
+
+1. Download the run_container.sh script corresponding to the release, see https://github.com/indigo-dc/Accounting/releases for a list of releases and corresponding docker image tag.
+
+2. Populate the following variables in `docker/run_container.sh`
+   ```
+   MYSQL_ROOT_PASSWORD: The database root password.
+
+   MYSQL_PASSWORD: The password for the APEL user.
 
-## Using the docker image
+   ALLOWED_FOR_GET: A (python) list of IAM service IDs allowed to submit GET requests.
 
-1. Install docker
+   SERVER_IAM_ID: A IAM ID corresponding to this instance, used to validate tokens.
 
-2. Download the corresponding run_container.sh script corresponding to the release.
+   SERVER_IAM_SECRET: A IAM secret corresponding to this instance, used to validate tokens.
 
-3. Populate the MYSQL variables and and IAM variables
+   DJANGO_SECRET_KEY: The Django server requires its own "secret".
+   ```
 
-4. Run `./run_container.sh indigo-dc/Accounting:X.X.X-X`
+3. Run `./run_container.sh indigo-dc/Accounting:X.X.X-X`
 
-5. Before the server will start, a certificate needs to be added to the container. Run `docker exec -it <docker_id> bash` to enter the container and then execute step 6 of the "Setup from source" instructions.
+4. Before the server will start, a certificate needs to be added to the container. This can be done by either modifying `./run_container.sh` to load the docker image with a certificate mounted into it, or by interacting with the image post start up with `docker exec -it <docker_id> bash`.
```

Comment:
I'd replace "post" with "after" for clarity. Don't want confusion with "POST". :grin: